### PR TITLE
Use absolute import

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

File: Sources/Filters/General/Calculator/index.js [hunk 1]
```diff
@@ -216,7 +208,7 @@ export function extend(publicAPI, model, initialValues = {}) {
 
 // ----------------------------------------------------------------------------
 
-export const newInstance = macro.newInstance(extend, 'vtkWarpScalar');
+export const newInstance = macro.newInstance(extend, 'vtkCalculator');
```
Comment (jourdain) [inline Sources/Filters/General/Calculator/index.js]: FYI @vibraphone

**Comment by jourdain** [comment]
FYI @martinken @MayeulChassagnard @fogleman @wschroed @vibraphone @scottwittenburg @aronhelser 

On top of fixing the import to be absolute, I fix some common mistake (me including) like:

1. Forgetting the registration of the constructor in the vtk factory by providing its name here:

```
export const newInstance = macro.newInstance(extend, 'vtkClassName__HERE');
``` 

2. Forgetting to register your class/module into and [index.js](https://github.com/Kitware/vtk-js/blob/c00d897b2c7c6fb1f0a563f62449db26aac3f297/Sources/Interaction/Manipulators/index.js)

File: Sources/Filters/General/Calculator/index.js (hunk 1 above)
Comment (vibraphone) [inline Sources/Filters/General/Calculator/index.js]: Oops, you have discovered my secret method for developing a new vtkjs object.

Comment (jourdain) [inline Sources/Filters/General/Calculator/index.js]: Don't worry I do the same thing. I also fixed some comments too, but at least the comments don't break anything... ;-)

**Pull request merged**